jest_config/jest.config.json

```diff
@@ -2,25 +2,27 @@
   "rootDir": "../",
   "verbose": true,
   "transform": {
-    "\\.tsx?$": "ts-jest"
+    "^.+\\.jsx?$": "babel-jest",
+    "^.+\\.tsx?$": "ts-jest"
   },
   "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
-  "moduleDirectories": ["node_modules", "common"],
+  "moduleDirectories": ["node_modules", "common"], 
   "moduleFileExtensions": ["ts", "tsx", "js", "jsx", "json", "worker.ts", "worker.tsx"],
   "moduleNameMapper": {
     "shared/(.*)": "<rootDir>/shared/$1",
+    "test-utils": "<rootDir>/jest_config/test-utils",
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/jest_config/__mocks__/fileMock.ts",
     "\\.(css|scss)$": "<rootDir>/jest_config/__mocks__/styleMock.ts",
     "\\.worker.(ts|tsx)": "<rootDir>/jest_config/__mocks__/workerMock.js"
   },
-  "testPathIgnorePatterns": ["node_modules", "dist"],
+  "testPathIgnorePatterns": [ "node_modules",  "dist" ],
```

Comment:
Can you remove the spaces here (to keep it consistent)?

Reply:
Done